### PR TITLE
Implement intro section

### LINKED ISSUE #14: (SP: 1)  Implement intro (hero) section
image

### PR DESCRIPTION
**Pull request by OliaHrytsak**
Added Banner section with using custom components and tests for it
Background color

image

### CODE REVIEW

File: code/src/layouts/app-banner/AppBanner.scss [hunk 1]
```diff
@@ -0,0 +1,26 @@
+.banner {
+  background-image: url("@/assets/images/home-page/hero_section_img.jpg");
+  background-size: cover;
+  background-position: center;
+  aspect-ratio: 12/5;
+
+  &__container {
+    max-width: 500px;
+    max-height: 400px;
+    height: auto;
```
Comment (mhevyk) [inline code/src/layouts/app-banner/AppBanner.scss]: It is redundant i think

File: code/src/layouts/app-banner/AppBanner.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    margin: 100px 50px;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
```
Comment (mhevyk) [inline code/src/layouts/app-banner/AppBanner.scss]: Same here

File: code/src/shared/modules/application/i18n/en.json [hunk 3]
```diff
@@ -1,3 +1,8 @@
 {
-  "application.header": "Hello, React with Webpack!!!"
-}
\ No newline at end of file
+  "application.header": "Hello, React with Webpack!",
+
+  "appBanner.header": "Welcome",
```
Comment (mhevyk) [inline code/src/shared/modules/application/i18n/en.json]: Remove space for better readability

File: code/src/shared/modules/application/i18n/uk.json [hunk 4]
```diff
@@ -1,3 +1,7 @@
 {
-  "application.header": "Привіт, React з Webpack!"
-}
\ No newline at end of file
+  "application.header": "Привіт, React з Webpack!",
+
```
Comment (mhevyk) [inline code/src/shared/modules/application/i18n/uk.json]: Same here

**Review by mhevyk — CHANGES_REQUESTED** [review]
Please fix following:
1. Change texts of hero section. It looks better if we have a few more words in title and description has less text, it looks more visually appealing for me
2. I suggest changing font size of description to 20px, it is `label` `AppTypography` variant

image

File: code/src/layouts/app-banner/AppBanner.scss (hunk 1 above)
Comment (OliaHrytsak) [reply code/src/layouts/app-banner/AppBanner.scss]: Fixed

File: code/src/layouts/app-banner/AppBanner.scss (hunk 2 above)
Comment (OliaHrytsak) [reply code/src/layouts/app-banner/AppBanner.scss]: Fixed

File: code/src/shared/modules/application/i18n/uk.json (hunk 4 above)
Comment (OliaHrytsak) [reply code/src/shared/modules/application/i18n/uk.json]: Fixed

**Comment by OliaHrytsak** [comment]
> Please fix following:
> 
> 1. Change texts of hero section. It looks better if we have a few more words in title and description has less text, it looks more visually appealing for me
> 2. I suggest changing font size of description to 20px, it is `label` `AppTypography` variant
> 
> image

image

**Review by D1nviz — APPROVED** [review]

**Review by mhevyk — APPROVED** [review]

**Review by dmtrth25 — APPROVED** [review]

**Pull request merged**